### ISSUE TITLE
UselessOverwrittenMethodViolation is not triggered without return

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-python-styleguide#1168]
# Bug report

## What's wrong

This code is considered valid:

```python
class Test:
    def test(self, a, b):
        super().test(a, b)
```

But, it should not. Because functions can (and usually do) return `None` and `return` is optional in this case.

@bellum you are a codeowner of this part. Was this intentional? Maybe I am missing something.

**Comment by bellum** [comment]
@sobolevn what if super().test(...) returns something? this overwrite can intentionally hide return value

**Comment by sobolevn** [comment]
Then it should be:

```python
class Test:
    def test(self, a, b):
        super().test(a, b)
        return None  # we hide return value, because inheritance sucks!
```

It is way more readable. This case should go into the docs. Thanks a lot for the information!
